src/components/CloudinaryContext/CloudinaryContext.js

```diff
@@ -40,7 +42,10 @@ class CloudinaryContext extends CloudinaryComponent {
         allowedProps[currentProp] = this.props[currentProp];
         return allowedProps;
       }, {});
-    return <div {...nonCloudinaryProps}>{this.props.children}</div>;
+    const {includeOwnBody, ...props} = nonCloudinaryProps;
+    //{console.log(nonCloudinaryProps.includeonbody);}
+  
+  return nonCloudinaryProps.includeOwnBody ? this.props.children : <div {...props}>{this.props.children}</div>;  
```

Comment:
@strausr , in line 45 you extracted the value of nonCloudinaryProps.includeOwnBody into a variable, why not using it?